tests/unit/records/test_api_views.py

```diff
@@ -118,8 +120,12 @@ def test_custom_search_validation(es, api, json_headers, record_with_bucket,
 
 
 @pytest.mark.parametrize(('query', 'result'), [
-    ('[dwc:family]:Felidae', 1),
-    ('[dwc:family]:foobar', 0)
+    ('[dwc:family]:[Felidae]', 1),
+    ('[dwc:family]:[foobar]', 0),
+    ('[obo:RO_0002453]:[Cat:]', 1),
+    ('[obo:RO_0002453]:[:"Cat flea"]', 1),
+    ('[obo:RO_0002453]:[foobar:]', 0),
+    ('[obo:RO_0002453]:[(foobar OR "Felis catus"):]', 1),
```

Comment:
Minor: I might be mistaken but it seems to me that there is a test case missing here, for searching a specific relationship, providing both parts of it.